variables.tf

```diff
@@ -36,7 +36,7 @@ variable "asp_capacity" {
 variable "team_name" {
   type        = "string"
   description = "The name of your team"
-  default     = "CDM (CCD/DM)"
+  default     = "CCD"
```

Comment:
Should be CDM no?

Reply:
Probably but we're CCD everywhere else.  But CDM it shall be